package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vso-android-extension",
-  "version": "4.207.0",
+  "version": "4.208.0",
```

Comment:
Either let us update the version before the release of the extension, or update the version in _package-loc.json_ accordingly

Reply:
@LucaBlackDragon next steps...

Reply:
@max-zaytsev I have reverted this change too

Reply:
@max-zaytsev I think I should instead update the version in `task.json` and `task.loc.json`, am I right?

Reply:
@LucaBlackDragon  Yes, that's right